src/pages/_app.tsx

```diff
@@ -1,13 +1,20 @@
-import 'styles/index.scss'
-import '../../public/vendor/fontawesome-pro-5.15.1-web/css/all.min.css'
+import type { NextPage } from 'next'
```

Comment:
@abbyoung I also made some minor changes to this file based on the [FontAwesome docs around NextJS](https://fontawesome.com/v5.15/how-to-use/on-the-web/using-with/react#nextjs) which seemed like they might be more up to date than what we had in place!